extensions/lifecycle/LifecycleMetrics.js

```diff
@@ -0,0 +1,107 @@
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const LIFECYCLE_LABEL_ORIGIN =  'origin';
+const LIFECYCLE_LABEL_OP = 'op';
+const LIFECYCLE_LABEL_STATUS = 'status';
+
+const conductorLatestBatchTime = ZenkoMetrics.createGauge({
```

Comment:
time (usually) increases, is this true also for this timestamp?

If that is the case, we may use counter instead : so that prometheus gets the "intention" that this field increase monotonically, and handles reset appropriately.

Reply:
the timestamp is a  unix timestamp and will be a monotonically increasing number.